web-regresjonstest/src/test/kotlin/no/nav/su/se/bakover/web/komponenttest/SøknadsbehandlingAlderKomponentTest.kt

```diff
@@ -55,253 +55,259 @@ import vilkår.personligOppmøtevilkårInnvilget
 
 internal class SøknadsbehandlingAlderKomponentTest {
     @Test
-    fun `sanity check`() {
+    fun `skal ikke kunne sende inn alderssøknad`() {
         withKomptestApplication(
             clock = 17.juni(2022).fixedClock(),
         ) { appComponents ->
-            appComponents.services.søknad.nySøknad(
-                søknadInnhold = søknadsinnholdAlder(),
-                identBruker = saksbehandler,
-            )
 
-            val sak = appComponents.services.sak.hentSak(fnrOver67, Sakstype.ALDER).getOrFail()
-            val søknad = sak.søknader.single()
+            assertThrows<IllegalStateException> {
```

Comment:
Bare wrappet hele testen i denne. Vi kommer sikkert til å få bruk for den senere